libs/plugins/flow-client/src/lib/flow-diagram/flow-diagram.component.ts

```diff
@@ -50,6 +56,11 @@ export class FlogoFlowDiagramComponent {
       select(FlowSelectors.getSelectionForCurrentHandler),
       takeUntil(this.ngOnDestroy$)
     );
+    this.iconProvider$ = this.store.select(FlowSelectors.getCurrentItemsAndSchemas).pipe(
```

Comment:
nit: observable can be destroyed/unsubscribed 🤔

Reply:
I don't think it's needed because we never directly subscribe to it, the async pipe in the template handles the subscription and unsunscribes

Reply:
Okay, thought so.
Just that we are doing explicitly for other observables in this component so confused we may have to do it.